src/pages/home/Home.js

```diff
@@ -10,6 +10,7 @@ import LegalModal from './components/LegalModal';
 import Hero from './components/Hero';
 import HomeSection from './components/HomeSection';
 import SocialMedia from './components/SocialMedia';
+import ScrollTopButton from './components/ScrollTopButton';
```

Comment:
Should we move this component to `components`? I think we might want to use it somewhere else and not only on the home page, right?

Reply:
Sure! I'll move.

Reply:
Done!